src/workers/files.js

```diff
@@ -55,6 +58,11 @@ const processFile = async (key) => {
     });
     await updateFileStatus(key, APPROVED);
   } catch (error) {
+    // If the key couldn't be found, we can't update the status.
+    if (error.message === KEY_NOT_FOUND) {
+      throw error;
+    }
```

Comment:
Do we want to throw here rather than allow a re-try? The current code will attempt to retry if scanning is not rejected.

Reply:
Yeah, that's a good point. Updated here: 1dc7abf0c12830abdd1dbbc28fc27a8b21eca8c8